resources/waterheater.rb

```diff
@@ -588,24 +588,8 @@ def self.apply_indirect(model, runner, space, cap, vol, t_set, oncycle_p, offcyc
       tank_type = Constants.WaterHeaterTypeTank
       # Actual tank volume = 95% nominal tank volume
       act_vol = calc_storage_tank_actual_vol(vol, nil)
-      # Tank geometry
-      surface_area, a_side = calc_tank_areas(act_vol)
-
-      if standby_loss.nil? # Swiched to standby_loss equation fit from AHRI database
-        # calculate independent variable SurfaceArea/vol(physically linear to standby_loss/skin_u under test condition) to fit the linear equation from AHRI database
-        sqft_by_gal = surface_area / act_vol # sqft/gal
-        standby_loss = 2.9721 * sqft_by_gal - 0.4732 # linear equation assuming a constant u
```

Comment:
If someone were to mess up the inputs and get a sqft_by_gal value of < 0.159 (.4732/2.9721) you'd get standby loss < 0, which would cause the simulation to crash. If you do the math out for how surface area is calculated, there's probably a critical volume where the sign changes. I'd check for that case and throw an instructive error message about bad inputs to help users identify why the run crashed.

Reply:
Good point! I followed our area/volume calculation and solved this math, when act_v is over 787.8gal (that is, nominal volume over 787.8/0.95=829gal), the standby loss will be negative here. This value is obviously way too large than reality, I will add an error message if bad volume input is defined.